strutil/version.go

```diff
@@ -80,6 +73,37 @@ func chOrder(ch uint8) int {
 	return int(ch) + 256
 }
 
+func main() {
+	fmt.Println("var chOrder = [...]int{")
+	for i := 0; i < 16; i++ {
+		for j := 0; j < 16; j++ {
+			fmt.Printf("%d, ", chOrder(uint8(i*16+j)))
+		}
+		fmt.Println()
+	}
+	fmt.Println("}")
+}
+
+*/
+var chOrder = [...]int{
```

Comment:
Played with this a little and turned it into a generator in case it needs tweaking in the future. See https://github.com/snapcore/snapd/compare/master...bboozzoo:bboozzoo/chrorder-generator

Reply:
That's super nice, would you mind pushing the generator into this PR?